src/components/VideoPlayerContexts/PlaybackContext.js

```diff
@@ -0,0 +1,124 @@
+import PropTypes from 'prop-types';
+import React, {useCallback, useContext, useEffect, useMemo, useRef, useState} from 'react';
+import useCurrentReportID from '@hooks/useCurrentReportID';
+
+const PlaybackContext = React.createContext(null);
+
+function PlaybackContextProvider({children}) {
+    const [currentlyPlayingURL, setCurrentlyPlayingURL] = useState(null);
+    const [sharedElement, setSharedElement] = useState(null);
+    const [originalParent, setOriginalParent] = useState(null);
+    const currentVideoPlayerRef = useRef(null);
+    const {currentReportID} = useCurrentReportID();
+
+    const pauseVideo = useCallback(() => {
+        if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.setStatusAsync)) {
+            return;
+        }
+        currentVideoPlayerRef.current.setStatusAsync({shouldPlay: false});
+    }, [currentVideoPlayerRef]);
+
+    const stopVideo = useCallback(() => {
+        if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.stopAsync)) {
+            return;
+        }
+        currentVideoPlayerRef.current.stopAsync({shouldPlay: false});
+    }, [currentVideoPlayerRef]);
+
+    const playVideo = useCallback(() => {
+        if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.setStatusAsync)) {
+            return;
+        }
+        currentVideoPlayerRef.current.getStatusAsync().then((status) => {
+            const newStatus = {shouldPlay: true};
+            if (status.durationMillis === status.positionMillis) {
+                newStatus.positionMillis = 0;
+            }
+            currentVideoPlayerRef.current.setStatusAsync(newStatus);
+        });
+    }, [currentVideoPlayerRef]);
+
+    const unloadVideo = useCallback(() => {
+        if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.unloadAsync)) {
+            return;
+        }
+        currentVideoPlayerRef.current.unloadAsync();
+    }, [currentVideoPlayerRef]);
+
+    const updateCurrentlyPlayingURL = useCallback(
+        (url) => {
+            if (currentlyPlayingURL && url !== currentlyPlayingURL) {
+                pauseVideo();
```

Comment:
`pauseVideo` is being called even if the video context is already destroyed. This is causing error in console. ref: https://github.com/Expensify/App/issues/37142#issuecomment-1963050093